src/test/config.sol

```diff
@@ -1642,10 +1672,10 @@ contract Config {
             offboarding:  true
         });
         afterSpell.collaterals["PSM-PAX-A"] = CollateralValues({
```

Comment:
Offboarding flag is still `false` here.

If tests pass with it as `true` then do that (because it's the first PSM offboarding I don't know how it will go) but if tests start failing, let's talk about it.

Worst case we comment //true next to false and fix the tests when we actually offboard this, as this is just offboarding by definition and still only stage 1